### PR TITLE
Add more oidc scopes to navigator's configuration

### PR DESCRIPTION
**Pull request by grossvogel**
Please let me know if there's more I need to know about the packaging / publishing process (or anything else)

### CODE REVIEW

**Review by bit-herder — APPROVED** [review]
I think this is all that is required but lets get joel to look to be sure.

**Review by jwietelmann — APPROVED** [review]
In the future, don't run `./package.sh` on feature branches. (To avoid weird stuff happening when multiple PRs are open at a time, wait until after merge, pull master, bump the version, run `./package.sh`, then commit and push.)

File: charts/moondog-navigator/Chart.yaml
Comment (vaxinate) [inline charts/moondog-navigator/Chart.yaml:2]: Not sure on this. I know @jwietelmann has an opinion on `appVersion` vs `version` since usually the `appVersion` denotes the version of the application being installed by the chart, where `version` is the chart version.

**Review by vaxinate — APPROVED** [review]
looks fine. but maybe @jwietelmann should remind us what his opinion on bumping `appVersion` is

File: charts/moondog-navigator/Chart.yaml
Comment (grossvogel) [reply charts/moondog-navigator/Chart.yaml:2]: Oh, yeah... I've definitely been confused by that while reading up on other k8s apps

Comment (grossvogel) [reply charts/moondog-navigator/Chart.yaml:2]: Since navigator the app is still on 0.0.1 (and not yet public) I'm going to merge and let this discussion inform the _future_ process 😈

**Pull request merged**
